packages/lib/src/helpers/kv_public_input.ts

```diff
@@ -0,0 +1,173 @@
+var EC = require("elliptic").ec;
+const BN = require("bn.js");
+
+import { bytesToBigInt, bigIntToBytes } from "./utils";
+import { EffECDSAPubInput } from "../types";
+
+const ec = new EC("secp256k1");
+
+const SECP256K1_N = new BN(
+  "fffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141",
+  16
+);
```

Comment:
What does this do? just curiousity, i see it used down below, assuming something related to the curve math